Rtp/RtpFrame.cs

```diff
@@ -70,7 +70,7 @@ public static Common.MemorySegment AssemblePacket(RtpPacket packet, bool useExte
                 using (RtpExtension extension = packet.GetExtension())
                 {
                     //If present and complete
-                    if (Common.IDisposedExtensions.IsNullOrDisposed(extension).Equals(false) && extension.IsComplete)
+                    if (Common.IDisposedExtensions.IsNullOrDisposed(extension) is false && extension.IsComplete)
```

Comment:
[System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.AggressiveInlining)]
        public static bool IsNotNullOrDisposed(this IDisposed dispose) => IsNullOrDisposed(dispose) is false;